src/js/routes/legend/Legend.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import LegendContent from '../../../static/legend.html';
+
+const Legend = () => (
+	<div>
+		<div dangerouslySetInnerHTML={{ __html: LegendContent }} />
+	</div>
+);
+
+export default Legend;
```

Comment:
Completely understand, I'll update it. I'll search for the es-lint rule later today, or look at a custom validator.